src/main/java/mate/academy/rickandmorty/model/Character.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.rickandmorty.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "characters")
```

Comment:
to the future - conventionally we should name DB tables in singular, not plural:
```suggestion
@Table(name = "character")
```